### DISCUSSION TITLE
[REQUEST] searNXG privacy search engine LXC

### DISCUSSION THREAD

**Discussion by spants** [tteck/Proxmox#318] [Ideas]
This engine works really well as a docker but I would love to move to my Proxmox system. 
Online example site: pick from https://searx.space/
Documentation: https://docs.searxng.org
The installation instructions (over my head!) https://docs.searxng.org/admin/installation.html#installation-scripts

Thanks for your work on these scripts.

**Comment by mon5termatt** [comment]
bump

**Comment by ttJobo** [comment]
Yeah. I think Searxng script will be pretty cool alternative to the existing Whoogle.

**Comment by HerzogVonWiesel** [comment]
Bumping this as well!

**Comment by jedikaal** [comment]
I use SearXNG on a daily basis. Would like to see it as one of the script.

**Comment by halpcomputar** [comment]
One thing to keep in mind with private search engines: While they're awesome, you really only achieve anonymity if there are a lot of varied users of your instance. If you're the only one using your instance, your searches are going to be easily tracked. If you have the ability to host for a LOT of people, that's awesome. Otherwise, possibly consider contributing to a larger provider of a search instance you like and trust.

**Reply by jedikaal** [reply]
Anonymity is possible with multiple user. But that is not the only feature. SearNXG can search in multiple sources all at the same with and without advertisements, promoted results etc. While having one interface, you get result from a large number of databases if you choose so. It is very customizable.

**Reply by halpcomputar** [reply]
Agreed 100% on all fronts. I just think it's important to point out that by self hosting and being the only user, you're actually damaging privacy. SearXNG is my go-to, and I've been using it since it was just SearX - I just use larger hosted instances for the reasons mentioned when privacy matters.

**Comment by Bosnis** [comment]
Did this ever happen?

**Comment by pestywanderer** [comment]
Bump to this, I really would love to see this as a script in the repo!

**Comment by jotenakis2** [comment]
Would be really great

**Comment by budney76** [comment]
Not a fan of Docker, so this would awesome to see!

**Comment by MickLesk** [comment]
Feedback from me (as a contributor):
The documentation of SearXNG is so shitty that you can't create a proper script with it. Not even in normal deploys on a Debian LXC. 

It feels like half of it is missing, several things do not work.
If someone has the time and desire at some point, they can give it a try. I gave up after a few hours.

**Reply by Vakrehus** [reply]
I have created a script. Not as nice as tteck's, but it works for now.

Create a LXC with debian 12 with the script from tteck: https://tteck.github.io/Proxmox/#debian-lxc

When this is running, go into the LXC and execute my script:

`bash -c “$(wget -qLO - https://raw.githubusercontent.com/Vakrehus/pr0xm0x/main/searxng-lxc-install.sh)”
`

It is best to leave everything on default.
Have fun
